gcloud-java-pubsub/src/main/java/com/google/gcloud/pubsub/testing/LocalPubsubHelper.java

```diff
@@ -0,0 +1,83 @@
+package com.google.gcloud.pubsub.testing;
+
+import com.google.gcloud.pubsub.spi.testing.LocalPublisherImpl;
+import com.google.pubsub.v1.PublisherGrpc;
+
+import io.grpc.ManagedChannel;
+import io.grpc.Server;
+import io.grpc.netty.NegotiationType;
+import io.grpc.netty.NettyChannelBuilder;
+import io.grpc.netty.NettyServerBuilder;
+import io.netty.channel.local.LocalAddress;
+import io.netty.channel.local.LocalChannel;
+import io.netty.channel.local.LocalServerChannel;
+
+import java.io.IOException;
+import java.net.SocketAddress;
+
+/**
+ * A class that runs an in-memory Publisher instance for use in tests.
+ */
+public class LocalPubsubHelper {
+  private static final SocketAddress address = new LocalAddress("in-process-1");
```

Comment:
The comment made earlier (for PublisherApiTest) by @mziccard was not addressed.
"Can we make new LocalAddress("in-process-1") a private static final field?"